test/src/e2e_vm_tests/test_programs/should_fail/const_nonconst_init/test.toml

```diff
@@ -0,0 +1,4 @@
+category = "fail"
+
+# check: $()const X = bla(0);
+# check-next: $()Could not evaluate initializer to a const declaration.
```

Comment:
`check-next` isn't a thing with the `FileCheck` crate -- that's from LLVM `FileCheck` 🙂.  You probably want `nextln`.  Unfortunately this would've just been ignored and isn't an error.